include/json/reader.h

```diff
@@ -392,6 +392,12 @@ bool JSON_API parseFromStream(CharReader::Factory const&, IStream&, Value* root,
  */
 JSON_API IStream& operator>>(IStream&, Value&);
 
+/** Get the line and column of a character within a string.
```

Comment:
It feels like a different general function that has nothing to do with JSON.

This isn't part of Reader's API, so I'm not sure it belongs in reader.h.

The out parameters should be a returned struct rather than int references, and the location should be a `size_t`, as it's encoding an array offset.

The documentation should be more complete about whether the line and column are 1 or 0 based, how it handles newlines, and how it handles unicode multibyte chars. I mean is 八 counted as one character or are all of its utf-8 bytes counted? If you're doing line and column, then you're implicitly talking about display position and it's a can of worms.

But really I have to question its role in the API. It doesn't really fit in.